File: angular_workspace/pfiva-web/src/app/data-model/snips/Intent.ts
export class Intent {
    private _intentId: number;
    private _intentName: string;
    private _probability: number;

	public get intentId(): number {
		return this._intentId;
    }
    
	public set intentId(value: number) {
		this._intentId = value;
	}

	public get intentName(): string {
		return this._intentName;
	}

	public set intentName(value: string) {
		this._intentName = value;
	}

	public get probability(): number {
		return this._probability;
    }
    
	public set probability(value: number) {
		this._probability = value;
	}
}

File: workspace/data-model/src/main/java/de/pfiva/data/model/snips/Intent.java
package de.pfiva.data.model.snips;

import java.io.Serializable;
import java.math.BigDecimal;

public class Intent implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int intentId;
	private String intentName;
	private BigDecimal probability;
	
	public int getIntentId() {
		return intentId;
	}
	public void setIntentId(int intentId) {
		this.intentId = intentId;
	}
	public String getIntentName() {
		return intentName;
	}
	public void setIntentName(String intentName) {
		this.intentName = intentName;
	}
	public BigDecimal getProbability() {
		return probability;
	}
	public void setProbability(BigDecimal probability) {
		this.probability = probability;
	}
	
	@Override
	public String toString() {
		return "Intent [intentId=" + intentId + ", intentName=" + intentName + ", probability=" + probability + "]";
	}
}
